### DISCUSSION TITLE
Introduce hideFromInspector prop for queries run with DataSourceWithBackend's query method

### DISCUSSION THREAD

**Discussion by ivanahuckova** [grafana/grafana#51879] [Platform]
**What happened**:

When running queries using `getBackendSrv().fetch` we are able to use `hideFromInspector` [prop to not include query call in query inspector](https://github.com/grafana/grafana/blob/main/packages/grafana-runtime/src/services/backendSrv.ts#L52).  This is very useful in cases, where we run additional queries (e.g. log volume query or get log sample query in Loki), but we don't want to show this queries in Inspector. 

**What you expected to happen**:

Have an option to use `hideFromInspector` prop in the same way as it is when running queries using `getBackendSrv().fetch`

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
